sdk/template/azure-sdk-template/pom.xml

```diff
@@ -27,6 +27,16 @@
     <url>https://github.com/Azure/azure-sdk-for-java</url>
   </scm>
 
+  <properties>
+    <!-- Once full testing is added this should either be removed or increased -->
+    <!-- By default the targets are 0.40 and 0.30 respectively -->
+    <jacoco.min.linecoverage>0.10</jacoco.min.linecoverage>
+    <jacoco.min.branchcoverage>0.10</jacoco.min.branchcoverage>
```

Comment:
Let's remove this as well. Don't want to set the bar low to start with.

Reply:
I'd rather leave this as it makes on-boarding easier and a clear expectation that it will be removed once tests are implemented